opensearch/connection/base.py

```diff
@@ -127,6 +131,29 @@ def __init__(
         if opaque_id:
             self.headers["x-opaque-id"] = opaque_id
 
+        if (
+            os.getenv("ELASTIC_CLIENT_APIVERSIONING") == "1"
+            and os.getenv("OPENSEARCH_CLIENT_APIVERSIONING") == "1"
+        ):
+            raise RuntimeError(
+                "Please specify only one of "
+                "OPENSEARCH_CLIENT_APIVERSIONING or "
+                "ELASTIC_CLIENT_APIVERSIONING environment variable's value to "
+                "'1'"
+            )
+
+        if os.getenv("ELASTIC_CLIENT_APIVERSIONING") == "1":
+            self.headers.setdefault(
+                "accept", "application/vnd.elasticsearch+json;compatible-with=7"
```

Comment:
Could `"application/vnd.elasticsearch+json"` and `"application/vnd.opensearch+json"` be constants? They are reused  below.

Reply:
Could be done. Will re-visit when we pick up work on this.